WORKSPACE

```diff
@@ -90,18 +90,30 @@ npm_translate_lock(
         "//npm/private/test/vendored/semver-max:package.json",
     ],
     generate_bzl_library_targets = True,
-    lifecycle_hooks_exclude = [
+    lifecycle_hooks = {
+        # we fetch @kubernetes/client-node from source and it has a `prepare` lifecycle hook that needs to be run
+        "@kubernetes/client-node": [
+            "prepare",
+            "preinstall",
+            "install",
+            "postinstall",
+        ],
         # 'install' hook fails as it assumes the following path to `node-pre-gyp`: ./node_modules/.bin/node-pre-gyp
         # https://github.com/stultuss/protoc-gen-grpc-ts/blob/53d52a9d0e1fe3cbe930dec5581eca89b3dde807/package.json#L28
-        "protoc-gen-grpc@2.0.3",
-    ],
+        "protoc-gen-grpc@2.0.3": [],
+    },
     lifecycle_hooks_execution_requirements = {
+        "*": [
```

Comment:
wdyt about calling this `default` rather than `*`? does that make it more clear that it won't be added to packages below? `*` matches everything but `default` applies if not overridden

Reply:
and obvs there had better not be https://npmjs.com/default

Reply:
that would be more of a breaking change since we already support "*" here

Reply:
Good APIs are hard build; especially when you'd trying not to make breaking changes on top of existing APIs